backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/InlineClasses.kt

```diff
@@ -39,6 +43,7 @@ fun IrType.getInlinedClassNative(): IrClass? = IrTypeInlineClassesSupport.getInl
  */
 fun IrType.isInlinedNative(): Boolean = IrTypeInlineClassesSupport.isInlined(this)
 fun IrClass.isInlined(): Boolean = IrTypeInlineClassesSupport.isInlined(this)
+fun IrClass.isPrimitiveType() = KonanPrimitiveType.byFqNameParts[packageFqName]?.get(name) != null
```

Comment:
The name might be ambiguous. Please use `isNativePrimitiveType` or `isPrimitiveTypeNative`.